src/home/components/components/pokemon-detail/redux/pokemonDetail.reducers.ts

```diff
@@ -0,0 +1,59 @@
+import {SAVE_POKEMON_DETAIL} from './pokemonDetail.actions'
+import {IAction} from '../../../../../reducers'
+import {createSelector} from 'reselect'
+import {homeBaseSelector} from '../../../../redux/home.reducers'
+import {IPokemonDetail, IPokemonSpirites, IPokemonStats, IPokemonTypes} from '../../../../PokemonInterfaces'
+
+export interface IPartialPokemonDetails {
```

Comment:
If you want to define a subset of an existing interface you can use [Pick](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys)

Or define a base interface then extend.